README.md

```diff
@@ -1 +1,50 @@
-# ToDoList
\ No newline at end of file
+# ToDoList
+
+> The repo contains my javascript code for the ToDo list project
+
+## Built With
+
+- Javascript
+- Html
+- Bootstrap
+
+[LiveLink](https://igorkol91.github.io/ToDoList/)
+
+
```

Comment:
### [Optional]
If you check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Setup`.

kindly add the following which is the minimum checklist for this README FILE for now;
- screenshot from your app.
- contact address, that is; Author
- setup 

Set up example:
```
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- run `npm i`
- run `npm start`
```